RustFxA/FirefoxAccountSignInViewController.swift

```diff
@@ -112,7 +115,12 @@ class FirefoxAccountSignInViewController: UIViewController, Themeable {
         button.addTarget(self, action: #selector(self.scanbuttonTapped), for: .touchUpInside)
     }
 
-    private lazy var emailButton: ResizableButton = .build { button in
+    private lazy var emailButton: SecondaryRoundedButton = .build { button in
+        let viewModel = SecondaryRoundedButtonViewModel(
+        title: .FxASignin_EmailSignin,
+        a11yIdentifier: AccessibilityIdentifiers.Settings.FirefoxAccount.fxaSignInButton
+        )
+        button.configure(viewModel: viewModel)
         button.layer.cornerRadius = UX.buttonCornerRadius
```

Comment:
Same comment as for `scanButton`. Some properties can be removed from here since they are defined in `SecondaryRoundedButton`

Reply:
Scan button still needs adjustments as commented